VideoWeb/VideoWeb/ClientApp/src/app/services/user-media-stream.service.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { UserMediaDevice } from '../shared/models/user-media-device';
-import { MockLogger } from '../testing/mocks/MockLogger';
+import { MockLogger } from '../testing/mocks/mock-logger';
```

Comment:
i don't really see the need for a mock console logger, just use the actual console logger itself imo

Reply:
i only renamed it to match our naming convention :P